### DISCUSSION TITLE
How to use an edge collection to link to multiple node/collection types?

### DISCUSSION THREAD

**Discussion by rxndxm** [TomGrozev/arangox_ecto#35] [Q&A] [answered]
@TomGrozev Hi! I'm looking to have a References edge, which can point to various Collections (for instance a Video can reference a Person or a Place) -- is this possible? I see only one Collection can be set in the _to: field:

``` 
use ArangoXEcto.Edge, from: User, to: Post
```

**Comment by TomGrozev** [comment, marked as answer]
At the moment this wouldn't be possible because this is used by Ecto to determine which schema something belongs to. However, I believe it would be possible since the fields are stored as the _id. So I propose that the _from and _to fields allow a list as an input. Would this suit your use case?

**Reply by rxndxm** [reply]
Yes - that'd be perfect :)

**Reply by rxndxm** [reply]
@TomGrozev is there a workaround I can use in the meantime that would give me the same functionality

**Reply by TomGrozev** [reply]
The only thing I can think of is using a common edge module.
